### ISSUE TITLE
force figure /table numbers

### ISSUE THREAD

**Issue by ValWood** [pombase/canto#1341]
Would anyone object if we did this.
It would make approval much easier.
And one day we might be able to  link to the images, who knows.

I always add them  pretty much (for phenotypes anyway, not for GO which are often not from one experiement)

**Comment by Antonialock** [comment]
maybe not for us?

**Comment by mah11** [comment]
tl;dr: I vote no.

Long-winded explanation: I have both practical and principled reasons not to like the idea. It would not be a problem to add a separate field explicitly for the figure or table number, but it shouldn't be required.

On principle, we should only make a field mandatory if it gathers an essential part of an annotation. As helpful as they are for us when we're doing approval, fig/table numbers are still just a convenience. They are not needed for an annotation to sense when it's displayed on a gene page or sent to downstream analysis. (Also, would we do any checking of the fig/t number entry itself? If yes, more work; if no, another small reason not to require it.)

In practice, it looks not-so-great for infrequent users and maddening for frequent users.

We have some community users who don't quite "get it", and submit sessions with very little of the possible annotations actually done. For those users, adding any mandatory field would be another barrier, so we shouldn't do it for anything that doesn't meet the essential-for-the-annotation criterion.

On the other hand, we have a growing cohort of users who already do sessions thoroughly and sometimes include figure or table numbers without prompting. If we add a field and Val asks people to use it, that group will respond well and fill it in most of the time even if it's not mandatory, because they like helping us. We'd get most of the benefit without any of the problems.

For frequent users like us, Canto is already very click-heavy and it would absolutely drive me round the bend to have another field to fill in every time. It would especially grate in the majority of sessions that we can and do just approve ourselves - for those papers it would be additional work for no gain.

**Comment by ValWood** [comment]
Will close this but please add figure numbers if poss...makes later fixes much easier and might be useful in the future...
